### ISSUE TITLE
swcarpentry/shell-novice-es

### ISSUE THREAD

**Issue by zkamvar** [carpentries/lesson-transition#81]
tracking issue for <https://github.com/swcarpentry/shell-novice-es>

Note: this will likely borrow from #80 

 - [x] fix math delimiters (fixed in https://github.com/swcarpentry/shell-novice-es/pull/68)

```
✔ Converting 'swcarpentry/shell-novice-es/_episodes/06-script.md' to sandpaper ... done

! Some math could not be parsed... likely because of shell variable examples
ℹ Below is the error
! Inline math delimiters are not balanced.

HINT: If you are writing BASIC code, make sure you wrap variable
      names and code in backtics like so: `INKEY$`.

Below are the pairs that were found:
                                                        start...end
                                                        -----...---
Las variables $1, $2 y $3 representan los argumentos para el ...
```

 - [x] fix definition list glossary 

```
learners/reference.md:31 [missing anchor]: [parámetro](#parmetro)
learners/reference.md:54 [missing anchor]: [directorio raíz](#directorio-raz)
learners/reference.md:63 [missing anchor]: [salida estándar](#salida-estndar)
learners/reference.md:74 [missing anchor]: [entrada estándar](#entrada-estndar)
learners/reference.md:75 [missing anchor]: [salida estándar](#salida-estndar)
learners/reference.md:84 [missing anchor]: [interfaz gráfica de usuario](#interfaz-grfica-del-usuario)
learners/reference.md:87 [missing anchor]: [interfaz de línea de comandos](#interfaz-de-lnea-de-comando)
learners/reference.md:102 [missing anchor]: [directorio raíz](#directorio-raz)
learners/reference.md:117 [missing anchor]: [interfaz de línea de comandos](#interfaz-de-lnea-de-comando)
learners/reference.md:126 [missing anchor]: [entrada estándar](#entrada-estndar)
```

**Comment by zkamvar** [comment]
this has been fixed in 6609413

**Comment by zkamvar** [comment]
There was an error in this transition, see: https://github.com/swcarpentry/shell-novice-es/pull/69#issuecomment-1534889522
